directory-listing/table-automode.css

```diff
@@ -0,0 +1,12 @@
+@import 'table.css';
+@media (prefers-color-scheme: dark) {
+  @import 'darkmode.css';

```

Comment:
Why not import `table-darkmode.css` here?

Reply:
Doing so would import `table.css` twice.  This probably isn't a major issue and I think your suggestion makes more sense from a maintenance perspective, but I think it should be noted.